osbrain/tests/test_nameserver.py

```diff
@@ -346,16 +349,17 @@ def test_random_nameserver_process():
         random_nameserver_process(port_start=22, port_stop=22, timeout=0.5)
 
 
+@skip_windows_port_reuse
 def test_nameserver_oserror(nsproxy):
     """
     Name server start() should raise an error if address is already in use.
     """
     with pytest.raises(RuntimeError) as error:
         run_nameserver(nsproxy.addr())
     assert 'OSError' in str(error.value)
-    assert 'Address already in use' in str(error.value)
```

Comment:
You removed this line but now that we are skipping for Windows, it should be there.